src/utils.ts

```diff
@@ -249,6 +250,37 @@ export async function getScores(
   scoreApiUrl = 'https://score.snapshot.org',
   options: any = {}
 ) {
+  if (!Array.isArray(addresses)) {
+    return Promise.reject('addresses should be an array of addresses');
+  }
+  if (addresses.length === 0) {
+    return Promise.reject('addresses can not be empty');
+  }
+  for (const address of addresses) {
+    if (!isValidAddress(address)) {
+      return Promise.reject(`Invalid address: ${address}`);
+    }
+  }
+  if (!isValidNetwork(network)) {
+    return Promise.reject(`Invalid network: ${network}`);
+  }
+  try {
+    strategies.forEach((strategy) => {
+      if (strategy.network && !isValidNetwork(strategy.network)) {
+        throw new Error(
+          `Invalid network (${strategy.network}) in strategy ${strategy.name}`
+        );
+      }
+    });
+  } catch (e) {
+    return Promise.reject(e);
+  }
```

Comment:
Why different approach here (try,catch), can be the same `Promise.reject` like every where else

Reply:
Because it can not reject inside forEach because of the anonymous function. Will use `.some` as suggested above

Reply:
```suggestion
  const invalidStrategy = strategies.find(strategy => strategy.network && !isValidNetwork(strategy.network));
  
  if (invalidStrategy) {
    return Promise.reject(new Error(`Invalid network (${invalidStrategy.network}) in strategy ${invalidStrategy.name}`));
  }
```